app/views/layouts/card.html.slim

```diff
@@ -1,6 +1,6 @@
 = content_for :content do
   .container.sm-pt4
-    div class="card #{yield(:card_cls)}"
+    div class="px2 py3 sm-py5 sm-px6 mx-auto mb5 border-box card #{yield(:card_cls)}"
```

Comment:
this is called atomic css! single purpose, utility classes -- the idea is that they should do one thing and do it well, should be simple and obvious to use, and should operate independently; background reading - http://mrmrs.io/writing/2016/03/24/scalable-css/ :)